lib/components/narrative/realtime-annotation.js

```diff
@@ -25,28 +27,35 @@ export default class RealtimeAnnotation extends Component {
     const innerContent = <div className='realtime-alert'>
       <div className='content'>
         <h3>
-          <i className='fa fa-exclamation-circle' /> Service update
+          <i className='fa fa-exclamation-circle' />
+          {' '}
+          <FormattedMessage id='components.RealtimeAnnotation.serviceUpdate' />
         </h3>
-        <p>
+        <p className='small'>
           {useRealtime
-            ? <span className='small'>
-              Your trip results have been adjusted based on real-time
-              information. Under normal conditions, this trip would take{' '}
-              <b>{coreUtils.time.formatDuration(realtimeEffects.normalDuration)} </b>
-              using the following routes:{' '}
-              {filteredRoutes
-                .map((route, idx) => (
-                  <span key={idx}>
-                    <b>{route}</b>
-                    {filteredRoutes.length - 1 > idx && ', '}
-                  </span>
-                ))
-              }.
-            </span>
-            : <span className='small'>
-              Your trip results are currently being affected by service delays.
-              These delays do not factor into travel times shown below.
-            </span>
+            ? (
+              <FormattedMessage
+                id='components.RealtimeAnnotation.delaysShownInResults'
+                values={{
+                  b: chunks => <b>{chunks}</b>,
+                  normalDuration: (
+                    <FormattedDuration
+                      duration={realtimeEffects.normalDuration}
+                      // FIXME: move id and zeroHoursId inside component.
+                      id='common.time.tripDurationFormat'
+                      zeroHoursId='common.time.tripDurationFormatZeroHours'
```

Comment:
Is this fixme still relevant?

Reply:
Good point, its no longer needed

Reply:
Addressed this FIXME in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/ad8ea3326848a5ac077e01898d7f7657412fda84